src/package-scripts.js

```diff
@@ -1,7 +1,7 @@
 const lintDirs = 'src/ test/';
 module.exports = {
-  'test': 'istanbul cover test/', // eslint-disable-line
-  'postest': 'istanbul check --statements 90 --functions 90 --branches 90', // eslint-disable-line
+  'test': 'nyc node test/', // eslint-disable-line
+  'postest': 'nyc check-coverage --lines 90 --functions 90 --branches 90', // eslint-disable-line
```

Comment:
no formal way.  we could bump to 100, but most of the time the trickling single digit %s are error cases that we didn't want to simulate (perhaps a weird HTTP response code), or error conditions that are difficult to replicate.  it's easy to ignore those from coverage, but perhaps not always beneficial.  once in the habit of getting your coverage to a 90+ number anyway, you'll often strive for as high as you can without wasting time on obvious fringe cases